src/index.js

```diff
@@ -18,7 +19,9 @@ root.render(
         <BrowserRouter>
             <AuthContextProvider>
                 <NoteContextProvider>
-                    <App />
+                    <FilterContextProvider>
+                        <App />
+                    </FilterContextProvider>
```

Comment:
Instead of having multiple providers in your index.js file you can create a new file known as CombinedProviders.jsx 
```
import React from "react";
import { AuthProvider, NotesProvider } from "LOCATION WHERE CONTEXTS ARE STORED";

const CombinedProvider = ({ children }) => {
    return (
        <AuthContextProvider>
            <NoteContextProvider>
                <FilterContextProvider>
                {children}
                </FilterContextProvider>
            </NoteContextProvider>
        </AuthContextProvider>
    )
}

export { CombinedProvider };

```
And then in your index.js file you can use this CombinedProvider